Project/core/src/main/java/View/screen/GameScreenComponents/MoveButton.java

```diff
@@ -0,0 +1,75 @@
+package View.screen.GameScreenComponents;
+
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+
+import Controller.CheckAction;
+import Controller.GameController;
+import Controller.RecruiterActionController;
+import Model.AbstractCell;
+import Model.NormalCell;
```

Comment:
I think these imports can be removed if the cell is removed (see below suggestions).